test/models/ModeSpec.scala

```diff
@@ -14,16 +14,15 @@
  * limitations under the License.
  */
 
-package views.helpers
+package models
 
-import play.api.i18n.Messages
+import base.SpecBase
 
-object PageTitle {
```

Comment:
This is used by the layout.scala.html
Will this not effect functionality by deleting it?

Reply:
pageTitle is an argument in layout.scala.html
i've checked it before deleting and it was alright.